guides/plugins/plugins/content/seo/extend-robots-txt.md

```diff
@@ -0,0 +1,288 @@
+---
+nav:
+  title: Extend robots.txt configuration
+  position: 20
+
+---
+
+# Extend robots.txt configuration
+
+## Overview
+
+Since Shopware 6.7.1, the platform provides full `robots.txt` support with all standard directives and user-agent blocks. This feature was developed as an open-source contribution during October 2024 ([learn more](https://www.shopware.com/en/news/hacktoberfest-2024-outcome-a-robots-txt-for-shopware/)). For general configuration, refer to the [user documentation](https://docs.shopware.com/en/shopware-6-en/tutorials-and-faq/creation-of-robots-txt).
+
+Starting with Shopware 6.7.5, you can extend the `robots.txt` functionality through events to:
+
+* Add custom validation rules during parsing
+* Modify or generate directives dynamically
+* Support custom or vendor-specific directives
+* Prevent warnings for known non-standard directives
+
+::: info
+The events described in this guide require Shopware 6.7.5 or later.
+:::
```

Comment:
```suggestion
::: info
The events and features described in this guide are available since Shopware 6.7.5.
:::

You can extend the `robots.txt` functionality through events to:

* Add custom validation rules during parsing
* Modify or generate directives dynamically
* Support custom or vendor-specific directives
* Prevent warnings for known non-standard directives
```